source/numerics.tex

```diff
@@ -9029,519 +9034,370 @@
   constexpr float truncf(float x);
   constexpr long double truncl(long double x);
 
-  constexpr float fmod(float x, float y);       // see \ref{library.c}
-  constexpr double fmod(double x, double y);
-  constexpr long double fmod(long double x, long double y);     // see \ref{library.c}
+  constexpr @\placeholder{floating-point-type}@ fmod(@\placeholder{floating-point-type}@ x, @\placeholder{floating-point-type}@ y);
   constexpr float fmodf(float x, float y);
   constexpr long double fmodl(long double x, long double y);
 
-  constexpr float remainder(float x, float y);  // see \ref{library.c}
-  constexpr double remainder(double x, double y);
-  constexpr long double remainder(long double x, long double y);    // see \ref{library.c}
+  constexpr @\placeholder{floating-point-type}@ remainder(@\placeholder{floating-point-type}@ x, @\placeholder{floating-point-type}@ y);
   constexpr float remainderf(float x, float y);
   constexpr long double remainderl(long double x, long double y);
 
-  constexpr float remquo(float x, float y, int* quo);   // see \ref{library.c}
-  constexpr double remquo(double x, double y, int* quo);
-  constexpr long double remquo(long double x, long double y, int* quo);     // see \ref{library.c}
+  constexpr @\placeholder{floating-point-type}@ remquo(@\placeholder{floating-point-type}@ x, @\placeholder{floating-point-type}@ y, int* quo);
   constexpr float remquof(float x, float y, int* quo);
   constexpr long double remquol(long double x, long double y, int* quo);
 
-  constexpr float copysign(float x, float y);   // see \ref{library.c}
-  constexpr double copysign(double x, double y);
-  constexpr long double copysign(long double x, long double y);     // see \ref{library.c}
+  constexpr @\placeholder{floating-point-type}@ copysign(@\placeholder{floating-point-type}@ x, @\placeholder{floating-point-type}@ y);
   constexpr float copysignf(float x, float y);
   constexpr long double copysignl(long double x, long double y);
 
   double nan(const char* tagp);
   float nanf(const char* tagp);
   long double nanl(const char* tagp);
 
-  constexpr float nextafter(float x, float y);      // see \ref{library.c}
-  constexpr double nextafter(double x, double y);
-  constexpr long double nextafter(long double x, long double y);    // see \ref{library.c}
+  constexpr @\placeholder{floating-point-type}@ nextafter(@\placeholder{floating-point-type}@ x, @\placeholder{floating-point-type}@ y);
   constexpr float nextafterf(float x, float y);
   constexpr long double nextafterl(long double x, long double y);
 
-  constexpr float nexttoward(float x, long double y);       // see \ref{library.c}
-  constexpr double nexttoward(double x, long double y);
-  constexpr long double nexttoward(long double x, long double y);   // see \ref{library.c}
+  constexpr @\placeholder{floating-point-type}@ nexttoward(@\placeholder{floating-point-type}@ x, @\placeholder{floating-point-type}@ y);
```

Comment:
This was the result of a typo when I was doing the mass change to the `<cmath>` synopsis.  The signature of `nexttoward` should be 

`constexpr @\placeholder{floating-point-type}@ nexttoward(@\{placeholder{floating-point-type}@ x, long double y);`

This breaking design change (changing the type of the second parameter of `nexttoward`) was not in the version of the paper that was approved by LEWG.  The change was accidentally introduced without any comment or discussion and without anyone noticing during processing by LWG.

I could fix this with a NB comment.  But I think the mistake is obvious enough that it could be fixed here before the CD is finalized, especially since the author of the mistake swears that it was an honest mistake that no one noticed during the review.

Reply:
The previous comment misses that a paper is no longer "the author's" once is was approved by the various WG21 stages. @tkoeppe , for you to decide.

Reply:
Please file an LWG issue for this asap. If we can get LWG sign-off that this was an obvious oversight that can be fixed editorially ("because the intention was clear"), then maybe we can fix this for the CD, otherwise we'll have to fix it through the regular issues process. I'm not sure you even need an NB comment, since we have a standing comment to "fix all issues" anyway, but we should make sure to keep track of this.

Reply:
BTW @tkoeppe : Is it possible to generate a public accessible link to a PDF of the working paper in flight for issues like this one, whose contents cannot reasonably be delta-ed against N4910 and provide this link to me?

Reply:
@Dani-Hub Hi, thanks! I was just going to contact you and mention the uploaded artefacts we provide for _every_ precommit check. For example, for Motion 13, see https://github.com/cplusplus/draft/actions/runs/2806916819, and download "draft-snapshot". In general, for any commit with a precommit check, click the green tick to see the action run, then go to "Summary", and you should find the PDF there.

Does that help/work for you?

Reply:
Yes, thank you very much.

Reply:
My great pleasure!